### PR TITLE
Column number breakpoint

### PR DESCRIPTION
**Pull request by roieu**
Associated Issue: #2864 

Here's the Pull Request Doc
https://devtools-html.github.io/debugger.html/CONTRIBUTING.html#pull-requests

### Summary of Changes

* Added a column line to breakpoint label (only for column breakpoint)

### Test Plan

Added column breakpoint and line breakpoint. Made sure everything's working super.

Example test plan:

- [x] Command-P opens the panel
- [x] Clicking “+” opens the panel
- [x] Clicking a source navigates to the source
- [x] Clicking "x" closes the panel

Here's the Debugger's Testing doc
https://docs.google.com/document/d/1oBMRxV8A2ag2t22YsQOxTdEv0mXKzIg0tggJjRkU1S0/edit#.
Feel free to improve it!

### Screenshots/Videos (OPTIONAL)

image

### CODE REVIEW

File: src/components/SecondaryPanes/Breakpoints.js [hunk 1]
```diff
@@ -37,11 +37,14 @@ function isCurrentlyPausedAtBreakpoint(pause, breakpoint) {
...
 
-function renderSourceLocation(source, line) {
+function renderSourceLocation(source, line, column) {
   const url = source.get("url") ? basename(source.get("url")) : null;
   // const line = url !== "" ? `: ${line}` : "";
   return url
-    ? dom.div({ className: "location" }, `${endTruncateStr(url, 30)}: ${line}`)
+    ? dom.div(
+        { className: "location" },
+        `${endTruncateStr(url, 30)}: ${line}${column ? `:${column}` : ""}`
```
Comment (jasonLaster) [inline src/components/SecondaryPanes/Breakpoints.js]: `column ? `:${column}` : ""` is a bit hard to follow. Can we pull this up to a local variable?

Comment (roieu) [inline src/components/SecondaryPanes/Breakpoints.js]: Yup. Fixing...

**Comment by roieu** [comment]
@jasonLaster check out the latest change.. LMK if there's anything else I need to fix
